backend/src/lang/lang.controller.ts

```diff
@@ -171,32 +181,41 @@ export class LangController {
       }
 
     } else if (intent.name === INTENT_HELP) {
+
       // Return Help
-      const history = this.dialogHistoryService.getHistory(uid);
-      const previousResponse = history[0];
-      const text = this.explanationService.getHelpText(previousResponse.intent, previousResponse.action);
-      return { text };
+      const history: Array<ConversationHistory> = await this.databaseLangService.getConversationHistoryOfUserWithoutIntents(uid,
```

Comment:
Ah hier werden die ignors spezifiziert?

Reply:
Korrekt. Für die Conversations Historie muss ich sie ja immer in die Datenbank schreiben, da sie dort auch angezeigt werden sollen. Bei der Abfrage des letzten Dialogflow Aufrufes suche ich dann aber eben nur Ereignisse, die weder INTENT_HELP noch INTENT_CONTEXT waren. Ich habe mir dabei auch erlaubt den default intent auszuschließen, da es ja passieren kann, dass Dialogflow einen Hilfe-Intent nicht zuordnen konnte ich aber eben Hilfe zu meiner letzten Thematik will.

Reply:
Top!